internal/pkg/sylog/sylog_test.go

```diff
@@ -0,0 +1,122 @@
+// Copyright (c) 2019, Sylabs Inc. All rights reserved.
+// This software is licensed under a 3-clause BSD license. Please consult the
+// LICENSE.md file distributed with the sources of this project regarding your
+// rights to use or distribute this software.
+
+// +build sylog
+
+package sylog
+
+import (
+	"strings"
+	"testing"
+)
+
+// TestSylogSetLevel to ensure the correct integer is returned
+func TestSylogSetLevel(t *testing.T) {
```

Comment:
None of these function names should be including `Sylog`. All of these functions are already within `package sylog`, so it's redundant